parodos-model-api/src/main/java/com/redhat/parodos/workflow/enums/LogType.java

```diff
@@ -0,0 +1,17 @@
+package com.redhat.parodos.workflow.enums;
+
+import lombok.Getter;
+
+public enum LogType {
+
+	TEXT(""), INFO("\u001B[32m"), WARNING("\u001B[33m"), ERROR("\u001B[34m"), END("\u001B[39m");
```

Comment:
It seems to me that we are mixing few things here. We have log levels (info, warn, err) and not log levels text and end. I think we should split

Reply:
> I do not really like the implementation TBH. As a user experience, I think that it's not great. For me, the implementation should be something.
> 
> this.log.error() this.log.info()
> 
> And execute context should be inherent inside the class, and the log should log the workflow.ID as wid='context.get(workflowid)' In this way, a user can retrieve all the logs related to a workflow based on a simple query in any 3-party tool (Loki, ElasticSearch, etc..)
> 
> Dancing the context with multiple functions is not the best, and a function like `addTaskLog` is not great for user experience.
> 
> But the code lgtm.

https://issues.redhat.com/browse/FLPATH-403